dockers/ubuntu-cuda/Dockerfile

```
@@ -20,7 +20,7 @@ ARG IMAGE_TYPE="devel"
 FROM nvidia/cuda:${CUDA_VERSION}-${IMAGE_TYPE}-ubuntu${UBUNTU_VERSION}
 
 ARG CUDNN_VERSION="9.1.0.70"
-ARG CUDNN_FRONTEND_CHECKOUT="v1.4.0"
+ARG CUDNN_FRONTEND_CHECKOUT="1.4.0"
```

```suggestion
ARG CUDNN_FRONTEND_VERSION="1.4.0"
```
Since it is version now, not building from git/source

---

done!